modules/cloudevent-recorder/bigquery.tf

```diff
@@ -99,3 +99,42 @@ resource "google_bigquery_data_transfer_config" "import-job" {
     delete_source_files             = false
   }
 }
+
+// Alert when no successful run in 30min, it should be successful every 15min
+resource "google_monitoring_alert_policy" "bq_dts" {
+  for_each = local.regional-types
```

Comment:
This could end up being a lot of alerts polluting the dashboard.

What if instead we used a `dynamic "conditions"` block below and use a single alert over all of the DTS configs?

Reply:
We could also do something like one per type, with a `dynamic` block over the regions

Reply:
agree, let me try to combine all the regions to a single alert.
would make it clearer in alerts what is failing